open-metadata-resources/open-metadata-deployment/docker/omrs-monitor/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project
+
+FROM node:10-alpine
```

Comment:
Nice move -- good low footprint. I tried some cut down distros but due to dependencies and time went with ubuntu on many - much larger footprint!